panoply/error_normalization.py

```diff
@@ -0,0 +1,97 @@
+from panoply import DataSource, PanoplyException
+from datetime import datetime
+
```

Comment:
Why isn't this just a part of the `errors.py` file or simply in an errors package? Internally you can organize it in many ways but why should the user be aware of the internal layout of this package?

The API of any package should be defined explicitly and it seems to me that like this we're "hiding" it in something that seems internal, but in fact isn't. I think it's fine to have a sub package for errors, but two is a bit confusing.

Reply:
Will create a sub package with errors.